slack_bolt/app/app.py

```diff
@@ -794,7 +792,7 @@ def __call__(*args, **kwargs):
 
     def function(
         self,
-        callback_id: Union[str, Pattern],
+        callback_id: Union[str, Function],
```

Comment:
Yeah, I agree that we don't need to support regexp here

Reply:
Yess but I think Function might be a typo, Ill look into this